scripts/smtk2ssrf-build.sh

```diff
@@ -108,15 +108,16 @@ else
 		cd "$BASEDIR"/mdbtools || aborting "Couldn't cd into $BASEDIR/mdbtools"
 		git pull --rebase || aborting "Problem downloading/updating mdbtools"
 	else
-		git clone https://github.com/brianb/mdbtools.git "$BASEDIR"/mdbtools || \
+		git clone https://github.com/mdbtools/mdbtools.git "$BASEDIR"/mdbtools || \
 			aborting "Problem downloading/updating mdbtools"
 	fi
 
 	echo "----> Building mdbtools ..."
 	echo "----> This will display a lot of errors and warnings"
 	cd "$BASEDIR"/mdbtools || aborting "Couldn't cd into $BASEDIR/mdbtools"
+	git checkout "v1.0.0"
 	autoreconf -i -f
-	./configure --prefix "$INSTALL_ROOT" --disable-man --disable-gmdb2 >/dev/null
+	./configure --prefix "$INSTALL_ROOT" --disable-man --with-bash-completion-dir="$INSTALL_ROOT"/share/bash-completion/completions >/dev/null
```

Comment:
that seems just weird. Why would we set a bash completion dir?

Reply:
Some time ago, this version of mdbtools refused to build unless this was explicitly set.  Probably a debian artifact.  Now it builds just fine without it, so I just force pushed the modification to the PR removing the bash-completion thing.

Reply:
Sorry, new force push.  The bash completion thing is set in configure.ac in mdbtools 
and must be turned off explicitly.